app/presenters/hyrax/collapsable_section_presenter.rb

```diff
@@ -0,0 +1,50 @@
+module Hyrax
+  # Draws a collapsable list widget using the Bootstrap 3 / Collapse.js plugin
+  class CollapsableSectionPresenter
+    def initialize(view_context, text, id:, icon_class:, open:)
```

Comment:
Make both `view_context` and `text` keywords.

Is there a reasonable default assumption for `open`